docs/partial_source/deep_dive/ivy_frontends_tests.rst

```diff
@@ -490,6 +490,61 @@ This function requires us to create extra functions for generating :code:`shape`
 * We use :code:`helpers.get_dtypes` to generate :code:`dtype`, these are valid numeric data types specifically for Torch.
 * :func:`torch.full` supports :code:`out` so we generate :code:`with_out`.
 
+Testing Without Using Tests Values
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+While even using hypothesis, there are some cases in which we do :code:`test_values=False` for example, we have a
+function add_noise() and we call it on x and we try to assert (we interally use assert np.all_close) that the result
+from torch backend matches tensorflow the test will always fail, because the function add_noise() depends on a random
+seed internally that we have no control over, what we change is only how we test for equality, in which in that case
+we can not and we have to reconstruct the output as shown in the example below.
```

Comment:
We should also explain the example of how we test these cases, the following example may be a bit **hard** to understand without explanation, as we still calling `assert_all_close`, which may be confusing for the reader. I'd suggest also adding a very simplified example, does not have to be in the codebase, just explain how we can check for size of the output, dtype, etc.